livekit-agents/livekit/agents/worker.py

```diff
@@ -683,15 +696,29 @@ def _on_process_job_launched(self, proc: ipc.job_executor.JobExecutor) -> None:
         self._update_job_status_sync(proc)
 
     async def _update_worker_status(self):
+        job_cnt = len(self._proc_pool.get_running_jobs())
         if self._draining:
-            update = agent.UpdateWorkerStatus(status=agent.WorkerStatus.WS_FULL)
+            update = agent.UpdateWorkerStatus(
+                status=agent.WorkerStatus.WS_FULL, job_count=job_cnt
+            )
             msg = agent.WorkerMessage(update_worker=update)
             await self._queue_msg(msg)
             return
 
-        current_load = await asyncio.get_event_loop().run_in_executor(
-            None, self._opts.load_fnc
-        )
+        def load_fnc():
+            signature = inspect.signature(self._opts.load_fnc)
+            parameters = signature.parameters
+            accepts_kwargs = any(
+                param.kind == inspect.Parameter.VAR_KEYWORD
+                for param in parameters.values()
+            )
+            if not accepts_kwargs and "job_cnt" not in parameters:
```

Comment:
We should not get the argument by name,  I don't think the argument should be a kwarg, but the worker would be the first positional one. I don't think we will add any other variables in the future.